traits/experimental/servicetrait/README.md

```diff
@@ -1,21 +1,25 @@
 # ServiceTrait
-You can use ServiceTrait to create a service for the workload on a Kubernetes cluster, like [ManualScalerTrait](https://github.com/oam-dev/spec/blob/master/core/traits/manual_scaler_trait.md).
+You can use ServiceTrait to create a k8s service for workload on a Kubernetes cluster
+
+Supported resources:
+- ContainerizedWorkload
+- StatefulSet
+- Deployment
 # Getting started
 - At first, you should follow [addon-oam-kubernetes-local](https://github.com/crossplane/addon-oam-kubernetes-local). And install OAM Application Controller and OAM Core workload and trait controller.
-- Then, you should deploy a StatefulSetWorkload controller by following [statefulsetworkload](https://github.com/My-pleasure/statefulsetworkload#getting-started).
 - Get the Servicetrait project to your GOPATH
 ```
-git clone https://github.com/My-pleasure/statefulsetworkload.git
+git clone https://github.com/oam-dev/catalog.git
 ```
 - Fetch the servicetrait image
 ```
-docker pull chienwong/servicetrait:v0.6
+docker pull chienwong/servicetrait:v1.6
```

Comment:
It would be better if you explain a little big how the image is build. Another way, replace it with `make docker-build` and change the name to more official like `oam-dev/servicetrait:xxx` in which situation you needn't involve your personal docker hub information:)